src/site/content/en/blog/nfc/index.md

```diff
@@ -263,11 +268,64 @@ const ndef = new NDEFReader();
 ndef.write("Writing data on an empty NFC tag is fun!", { overwrite: false })
 .then(() => {
   console.log("Message written.");
-}).catch(_ => {
+}).catch(error => {
   console.log(`Write failed :-( try again: ${error}.`);
 });
 ```
 
+### Make NFC tags read-only {: #make-read-only }
+
+To prevent malicious users from overwriting NFC tags content, it is possible to
+make NFC tags permanently read-only. This operation is a one-way process and
+cannot be reversed. Once an NFC tag has been made read-only, it can't be written
+anymore.
```

Comment:
Written TO

Reply:
Thanks! Fixed in https://github.com/GoogleChrome/web.dev/pull/7252/commits/0bfc261a0a5e472cde6c7b200842a7a206f33d0c